app/services/InUseMemoryService.scala

```diff
@@ -42,11 +42,41 @@ class InUseMemoryService extends InUseService {
 
   // stub data
 
+  val jsonDataOne = """{"category" : "Test Cat One",
+      |                 "user" : "Test User One",
+      |                 "target" : "Test Target One",
+      |                 "datetime" : "Test Date One",
+      |                 "function" : "Test Function One",
+      |                 "pkg" : "Test pkg One",
+      |                 "description" : "Description one in here",
+      |                 "asset" : "Test Asset One",
+      |                 "ip" : "xxx.xxx.xxx.xxx"}""".stripMargin
+
+  val jsonDataTwo = """{"category" : "Test Cat Two",
+                      | "user" : "Test User Two",
+                      | "target" : "Test Target Two",
+                      | "datetime" : "Test Date Two",
+                      | "function" : "Test Function Two",
+                      | "pkg" : "Test pkg Two",
+                      | "description" : "Description two in here",
+                      | "asset" : "Test Asset Two",
+                      | "ip" : "xxx.xxx.xxx.xxx"}""".stripMargin
+
+  val jsonDataThree = """{"category" : "Test Cat Three",
+                      | "user" : "Test User Three",
+                      | "target" : "Test Target Three",
+                      | "datetime" : "Test Date Three",
+                      | "function" : "Test Function Three",
+                      | "pkg" : "Test pkg Three",
+                      | "description" : "Description three in here",
+                      | "asset" : "Test Asset Three",
+                      | "ip" : "xxx.xxx.xxx.xxx"}""".stripMargin
+
   registerService("example service")
   registerService("empty service")
   registerService("another service")
-  registerCall("example service", ServiceCall("example service", DateTime.now.getMillis, "mock data one"))
-  registerCall("example service", ServiceCall("example service", DateTime.now.getMillis, "mock data two"))
-  registerCall("example service", ServiceCall("example service", DateTime.now.getMillis, "mock data three"))
+  registerCall("example service", ServiceCall("example service", DateTime.now.getMillis, jsonDataOne))
```

Comment:
Not neccesarily related to this PR, but these could probably live in ApplicationSpec, as they're related to testing. Feel free to add as a git issue, rather than making any changes in this PR as a bit out of scope